### DISCUSSION TITLE
Is there away to backup?

### DISCUSSION THREAD

**Discussion by Schizo-Rez** [k-ivan/visual-bookmarks-chrome#156] [General]
I'm looking for a way to backup my thumbnails, does anyone know how?

**Comment by k-ivan** [comment]
Hello @OnlyRez !
Unfortunately, this is not possible at the moment. I have been studying this possibility for a long time, but so far I do not have a solution to this issue.

**Reply by Schizo-Rez** [reply]
Got it, thank you anyway its such a good addon. Do you know where the images for the thumbnails get saved?

**Reply by k-ivan** [reply]
I am glad to help. In any case, I keep this question in mind.

You can get images from this folder(Win11):
```
C:\Users\{user_name}\AppData\Local\Google\Chrome\User Data\Default\IndexedDB\chrome-extension_jdbgjlehkajddoapdgpdjmlpdalfnenf_0.indexeddb.blob
```
